packages/launch-config/src/debug-config/config.ts

```diff
@@ -102,13 +100,13 @@ export function configureLaunchJsonFile(cwd: string, configOpts: DebugOptions):
     }
 
     // Add mock configuration for OData V2 or V4
-    if (odataVersion && [OdataVersion.v2, OdataVersion.v4].includes(odataVersion)) {
+    if (odataVersion && ['2.0', '4.0'].includes(odataVersion)) {
         const params = `${flpAppIdWithHash ?? ''}`;
         const mockCmdArgs =
-            isMigrator && odataVersion === OdataVersion.v2
+            isMigrator && odataVersion === '2.0'
```

Comment:
Separately, I think we need to look into this logic. 
Not all odata v2 apps have `test/FlpSandboxMockServer.html` which seems to be the assumption here.

Reply:
Thanks @devinea . Create a todo https://github.com/SAP/open-ux-tools/issues/2392 to address this. 👍